connector/countconnector/README.md

```diff
@@ -0,0 +1,213 @@
+# Count Connector
+
+| Status                   |                                                           |
+|------------------------- |---------------------------------------------------------- |
+| Stability                | [in development]                                          |
+| Supported pipeline types | See [Supported Pipeline Types](#supported-pipeline-types) |
```

Comment:
Perhaps we should establish a pattern for the headers of connectors? Like:

```suggestion
| Supported pipeline types as receiver | metrics |
| Supported pipeline types as exporter | traces, logs |
```

Reply:
I tried to come up with something that would fit this table's simple format but didn't like any of the options. Unfortunately we cannot just list the receiver and exporter types independently because support is defined by pairs of signals.

I considered using shorthand like `traces->metrics`, `metrics->metrics`, `logs->metrics`, but thought it was clearer to link to the other table.